packages/ember-htmlbars/tests/helpers/component_test.js

```diff
@@ -1,398 +0,0 @@
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /HTMLBars error: Could not find component named "does-not-exist"./, 'Expected missing component to generate an exception');
-});
-
-QUnit.test('component with unquoted param resolving to a component, then non-existent component', function() {
-  owner.register('template:components/foo-bar', compile('yippie! {{attrs.location}} {{yield}}'));
-  view = EmberView.create({
-    [OWNER]: owner,
-    dynamicComponent: 'foo-bar',
-    location: 'Caracas',
-    template: compile('{{#component view.dynamicComponent location=view.location}}arepas!{{/component}}')
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'yippie! Caracas arepas!', 'component was looked up and rendered');
-
-  run(function() {
-    set(view, 'dynamicComponent', undefined);
-  });
-
-  equal(view.$().text(), '', 'component correctly deals with falsey values set post-render');
-});
-
-QUnit.test('component with quoted param for non-existent component', function() {
-  view = EmberView.create({
-    [OWNER]: owner,
-    location: 'Caracas',
-    template: compile('{{#component "does-not-exist" location=view.location}}arepas!{{/component}}')
-  });
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /HTMLBars error: Could not find component named "does-not-exist"./);
-});
-
-QUnit.test('component helper properly invalidates hash params inside an {{each}} invocation #11044', function() {
-  owner.register('component:foo-bar', Component.extend({
-    willRender() {
-      // store internally available name to ensure that the name available in `this.attrs.name`
-      // matches the template lookup name
-      set(this, 'internalName', this.attrs.name);
-    }
-  }));
-  owner.register('template:components/foo-bar', compile('{{internalName}} - {{attrs.name}}|'));
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    items: [
-      { name: 'Robert' },
-      { name: 'Jacquie' }
-    ],
-    template: compile('{{#each view.items as |item|}}{{component "foo-bar" name=item.name}}{{/each}}')
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'Robert - Robert|Jacquie - Jacquie|', 'component was rendered');
-
-  run(function() {
-    set(view, 'items', [
-      { name: 'Max' },
-      { name: 'James' }
-    ]);
-  });
-  equal(view.$().text(), 'Max - Max|James - James|', 'component was updated and re-rendered');
-});
-
-QUnit.test('dashless components should not be found', function() {
-  expect(1);
-
-  owner.register('template:components/dashless', compile('Do not render me!'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{component "dashless"}}')
-  }).create();
-
-  expectAssertion(function() {
-    runAppend(view);
-  }, /You cannot use 'dashless' as a component name. Component names must contain a hyphen./);
-});
-
-QUnit.test('positional parameters does not clash when rendering different components', function(assert) {
```

Comment:
moved to `@htmlbars positional parameters does not clash when rendering different components`